packages/components/src/DataList/components/DataListLayout/DataListLayout.tsx

```diff
@@ -1,8 +1,18 @@
 import React from "react";
-import { DataListItemType, DataListObject } from "../../DataList.types";
+import {
+  Breakpoints,
+  DataListItemType,
+  DataListObject,
+} from "../../DataList.types";
 
 export interface DataListLayoutProps<T extends DataListObject> {
   children: (item: DataListItemType<T[]>) => JSX.Element;
```

Comment:
not on you, but forgot to readonly this
```suggestion
  readonly children: (item: DataListItemType<T[]>) => JSX.Element;
```